madhavtripathi05/main.dart

```diff
@@ -0,0 +1,9 @@
+void main(List<String> args) {
+  String profileUrl = 'https://github.com/madhavtripathi05';
+  var projectLinks = [
```

Comment:
Hey @madhavtripathi05 please add your code in here itself.

Reply:
okay will do